data/sql/updates/pending_db_world/rev_1745497441690295560.sql

```diff
@@ -0,0 +1,4 @@
+-- blue dragon 25% haste aura
+DELETE FROM `spell_script_names` WHERE `spell_id` = 45856;
+INSERT INTO `spell_script_names` (`spell_id`, `ScriptName`) VALUES
+(45856, 'spell_kiljaeden_breath_haste');
```

Comment:
```suggestion
DELETE FROM `spell_linked_spell` WHERE `spell_trigger` = 45856 AND `spell_effect` = 49725;
INSERT INTO `spell_linked_spell`
(`spell_trigger`, `spell_effect`, `type`, `comment`)
VALUES(45856, 49725, 1, 'Power of the Blue Flight - Breath: Haste');
```

https://www.azerothcore.org/wiki/spell_linked_spell

Reply:
Still needs the spell_script_names though correct?

Reply:
You can remove the spellscript and use only the linked spell entry in DB

Reply:
The checks didn't work without is why I asked. Made the edit and will wait for checks again

Reply:
I think you should remove all cpp fix. Only sql must remain.